pkg/virt-controller/services/template_test.go

```diff
@@ -4458,6 +4462,34 @@ var _ = Describe("Template", func() {
 			Expect(pod.Spec.Containers[0].Resources.Limits.Cpu().IsZero()).To(BeTrue())
 		})
 	})
+
+	Context("with serial console", func() {
+		DescribeTable("check for guest-console-log container", func(autoattachSerialConsole, logSerialConsole, expected bool) {
+			vmi := api.NewMinimalVMI("fake-vmi")
+			vmi.Spec.Domain.Devices.AutoattachSerialConsole = pointer.Bool(autoattachSerialConsole)
```

Comment:
```suggestion
			vmi.Spec.Domain.Devices.AutoattachSerialConsole = &autoattachSerialConsole
```

nit: Should be possible without `pointer` pkg. Same below.

Reply:
done